packages/aws-cdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Running the tests
+
+Unit tests are automatically run as part of the regular build. Integration tests
+aren't run automatically since they have nontrivial requirements to run.
+
+**CLI integration tests**
+
+CLI tests will exercise a number of common CLI scenario's, deploying actual
+stacks. To run, preload your shell with AWS credentials run run:
+
+```
+npm run integ-cli
```

Comment:
Isn't there a pre-requisite to have a fully built repo?

Reply:
No